src/writer.js

```diff
@@ -44,4 +44,35 @@ const Writer = M => {
 
 };
 
-module.exports = Writer;
\ No newline at end of file
+// Transformer
+Writer.WriterT = (Monoid, Monad) => {
+
+    const WriterT = daggy.tagged("run");
+
+    const W = Writer(Monoid);
+
+    WriterT[of] = (x) => WriterT(() => Monad[of](W[of](x)));
+
+    WriterT.prototype[map] = function(f){
+      return this[chain]((a) => WriterT[of](f(a)));
+    }
+
+    WriterT.prototype[chain] = function(f){
+      return WriterT(() => this.run()[chain]((outerWriter) => {
+        const outerTuple = outerWriter.run();
+        const newMonadWrappedWriter = f(outerTuple._1).run();
+        return newMonadWrappedWriter[map]((writer) => {
+          const innerTuple = writer.run();
+          return W(() => Tuple2(innerTuple._1, outerTuple._2.concat(innerTuple._2)));
+        })
+      }))
+    }
+
+    WriterT.prototype[ap] = function(b){
+      return this.chain(a => b.map(a))
```

Comment:
Ahha, had copied it from ```Writer.prototype.ap```. Fixing it in both places in f882192b428eefbb84cf24cd313bde4026d29447 (latest master/HEAD).